upf/upf_pfcp_api.c

```diff
@@ -1025,12 +1023,7 @@ handle_update_pdr (upf_session_t * sx, pfcp_update_pdr_t * update_pdr,
 
     if (ISSET_BIT (pdr->pdi.grp.fields, PDI_F_TEID))
       {
-	if (handle_f_teid
-	    (sx, gtm, &pdr->pdi, update, NULL, res, grp, 0) != 0)
-	  {
-	    r = -1;
-	    break;
-	  }
+	update->pdi.teid = pdr->pdi.f_teid;
```

Comment:
no, the `F-TEID` in a `Update PDR` can contain a reference to a CHID. That CHID must have been use in a `Create PDR` before and reference a valid TEID. 
The lookup part in handle_f_teid must still be executed, but should fail if doesn't find an existing TEID.